client/src/components/ElementEditor/containers/ElementEditor.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component, PropTypes } from 'react';
+
+class ElementEditor extends Component {
+  render() {
+    return (
+      <div />
+    );
+  }
+}
```

Comment:
I'm not 100% on the file paths here. I think I would've expected `client/src/components/ComponentName` or `client/src/containers/ContainerName`. What do you think? I realise that ElementEditor and HistoryViewer separation is a good idea though...

Reply:
I think it makes sense to keep the component domains separate (not sure if that's the right term but whatever). It feels slightly intense to have `import 'components/ElementEditor/containers/ElementEditor'` but I think it's better than combining `ElementalEditor and `HistoricElementView`.